tensorboard/backend/security_validator_test.py

```diff
@@ -40,48 +41,51 @@
 def create_headers(
     content_type="application/json",
     x_content_type_options="nosniff",
-    content_secrurity_policy=""
+    content_security_policy="",
 ):
   return Headers(
       {
           "Content-Type": content_type,
           "X-Content-Type-Options": x_content_type_options,
-          "Content-Security-Policy": content_secrurity_policy,
+          "Content-Security-Policy": content_security_policy,
       }
   )
 
 
 class SecurityValidatorMiddlewareTest(tb_test.TestCase):
   """Tests for `SecurityValidatorMiddleware`."""
 
-  def setUp(self):
-    super(SecurityValidatorMiddlewareTest, self).setUp()
-    self.get_header = lambda req: {}
-    app = werkzeug.Request.application(
-        lambda req: werkzeug.Response("OK", headers=self.get_header(req))
-    )
-    app = security_validator.SecurityValidatorMiddleware(app)
-    self.server = werkzeug_test.Client(app, werkzeug.BaseResponse)
-    self.mock_warn = mock.patch.object(logger, "warn").start()
-
-  def tearDown(self):
-    super(SecurityValidatorMiddlewareTest, self).tearDown()
-    mock.patch.stopall()
-
+  @mock.patch.object(logger, "warn")
```

Comment:
I still think if you're going to patch logger itself it would be best to scope it down as tightly as possible to just the call to `server.get("")` (by using a context manager).  Otherwise if anything in this method ends up emitting a warning, like the construction of `SecurityValidatorMiddleware`, it would fail the test.